default/tests/test_main.py

```diff
@@ -1,7 +1,7 @@
 from fastapi import FastAPI
 from fastapi.testclient import TestClient
 
-from default.main import app
+from default.main import app, datastore_client
```

Comment:
how does the normal datastore_client work when testing? does it connect to prod?